exec/java-exec/src/main/codegen/data/Parser.tdd

```diff
@@ -39,7 +39,13 @@
     "METADATA",
     "DATABASE",
     "IF",
-    "JAR"
+    "JAR",
+    "ANALYZE",
+    "COMPUTE",
+    "ESTIMATE",
+    "STATISTICS",
+    "SAMPLE",
+    "PERCENT"
```

Comment:
Adding so many "good" words as keywords will break existing queries:
```
SELECT estimate, percent FROM sample
```
Would, presumably, work prior to this commit, but will fail after. Or, were those keywords already reserved?

Do we have to consider backward compatibility here?